gems/aws-sdk-s3/lib/aws-sdk-s3/customizations/object.rb

```diff
@@ -492,6 +492,18 @@ def upload_file(source, options = {})
       #   retrieve the object. For more about object versioning, see:
       #   https://docs.aws.amazon.com/AmazonS3/latest/dev/ObjectVersioning.html
       #
+      # @option options [String] checksum_mode (ENABLED) When `ENABLED` and
```

Comment:
What are other values here and why is it a String instead of a boolean?

Reply:
ENABLED is actually the only value for it. It matches what is modeled on the api and leaves open the possibility of additional values in the future - I think we'd want to match the underlying api here in case this is added to in the future.

Reply:
Didn't know this was on the API. Makes sense